css/excercise_one/part-b/excercise_one.css

```diff
@@ -0,0 +1,318 @@
+* {
+  color: black;
+  font-family: 'Poppins',  sans-serif;
+  text-decoration: none;
+}
+
+.s-pt {
```

Comment:
let's try doing first the important thing, such as pt = padding-top and then the rest, so in this case it would be:
`pt-sm` I would also use sm for small instead of just the s, md for middle, lg for large, xl for extra large, this is what it's usually used